File: read/code.py
## DO NOT RUN
import os

os.chdir("read/")
##


import pandas as pd

pd.read_csv("data/data1.csv")

pd.read_csv("data-meta/data2.csv")
pd.read_csv("data-meta/data2.csv", skiprows=5)


pd.read_csv("data-meta-delim/data3.csv", skiprows=5)
pd.read_csv("data-meta-delim/data3.csv", skiprows=5, delimiter="$")

File: read/code.R
library(tidyverse)

read_csv('data/data1.csv')

read_csv('data-meta/data2.csv', skip = 6)

read_delim('data-meta-delim/data3.csv', skip = 6, delim = '$')

read_delim('data3-same.csv', skip = 6, delim = '$')

